first-responder/.htaccess

```
@@ -5,3 +5,4 @@ RewriteEngine on
 
 RewriteRule ^$ https://digitalbazaar.github.io/fr-spec/ [R=302,L]
 RewriteRule ^v1$ https://contexts.vcplayground.org/examples/first-responder/v1.json [R=302,L]
+RewriteRule ^sap/v1rc1$ https://digitalbazaar.github.io/fr-spec/contexts/sap-v1rc1.jsonld [R=302,L]
```

wildcard makes it easier to support new versions without changes here:
```suggestion
RewriteRule ^sap/v(.*)$ https://digitalbazaar.github.io/fr-spec/contexts/sap-v$1.jsonld [R=302,L]
```

---

Good note, thanks